content/applications/sales/sales/invoicing/time_materials.rst

```diff
@@ -2,135 +2,348 @@
+:guilabel:`Order Lines` tab (in the :guilabel:`Delivered` column) and in the new :guilabel:`Recorded
+Hours` smart button at the top of the top of the sales order.
+
+To invoice the customer for time spent on the project, click :guilabel:`Create Invoice`, and select
+:guilabel:`Regular invoice` from the :guilabel:`Create invoices` pop-up window. Then, click
+:guilabel:`Create Draft Invoice`.
+
+Doing so reveals a :guilabel:`Customer Invoice Draft`, clearly showing all the work that's been done
+in the :guilabel:`Invoice Lines` tab.
+
+.. tip::
+   Pay attention to the :guilabel:`Analytic Distribution` column in the :guilabel:`Invoice Lines`
+   tab, as that information is necessary to ensure other time/material invoicing tasks are completed
+   properly (and accurately).
+
+   .. image:: time_materials/invoice-lines-time.png
+      :align: center
+      :alt: Invoice draft showing time spent on sales order in Odoo Sales.
+
+To complete the invoice, click :guilabel:`Confirm`, and complete the invoicing process.
 
 Expenses configuration
 ======================
 
-To track & invoice expenses, you will need the expenses app. Go to
-:menuselection:`Apps --> Expenses` to install it.
+In order to track and invoice expenses related to a sales order, the Odoo *Expenses* app **must** be
+installed.
 
-You should also activate the analytic accounts feature to link expenses
-to the sales order, to do so, go to :menuselection:`Accounting -->
-Configuration --> Settings` and activate *Analytic Accounting*.
+To install the *Expenses* app, navigate to :menuselection:`Main Odoo dashboard --> Apps`. Then,
+on the :guilabel:`Apps` page, locate the :guilabel:`Expenses` app block, and click
+:guilabel:`Activate`. The page automatically refreshes and returns to the main Odoo dashboard, where
+the :guilabel:`Expenses` app is now available to access.
 
-Add expenses to your sales order
-================================
+Add expenses to sales order
+===========================
 
-From the expense app, you or your consultant can create a new one, e.g.
-the hotel for the first week on the site of your customer.
+To add an expense to a sales order, first navigate to the :menuselection:`Expenses` app. Then, from
+the main *Expenses* dashboard, click :guilabel:`New`, which reveals a blank expense detail form.
 
-You can then enter a relevant description and select an existing product
-or create a new one from right there.
+On the expense form, add a :guilabel:`Description` of the expense (e.g. Hotel Stay, Plane Ticket,
+etc.). Next, in the :guilabel:`Category` field, select the approrpiate option from the drop-down
+menu (e.g. Meals, Miles, Travel & Accomodation, etc.)
 
-.. image:: time_materials/time_materials06.png
+.. note::
+   Expense categories can be added and modified in the *Expenses* app by navigating to
+   :menuselection:`Expenses app --> Configuration --> Expense Categories`.
```

Comment:
If you want- I have a doc you can link here! The new expenses doc. It is still in review, but it would be great to add =)

Reply:
Right on. I'll be sure to keep this in mind as it makes its way through the review process.